Dockerfile.amd64

```diff
@@ -1,7 +1,7 @@
 FROM amd64/alpine:3.7
 MAINTAINER Tom Denham <tom@projectcalico.org>
 
-ADD dist/calicoctl-linux-amd64 /calicoctl
+ADD bin/calicoctl-linux-amd64 /calicoctl
```

Comment:
Thank you! I wanted to do this when I did the large cross-build and standardization PR, but thought it would raise too many hackles and mix too much together.